exec/java-exec/src/test/java/org/apache/drill/test/DrillTestWrapper.java

```diff
@@ -108,6 +110,10 @@
   // queries to run before the baseline or test queries, can be used to set options
   private String baselineOptionSettingQueries;
   private String testOptionSettingQueries;
+  // allow approximate equality tests for number types
+  private boolean approximateEquality;
```

Comment:
The notion of approximate equality and tolerance would be difficult to manage in a generalized functional testing environment.  If you want repeatability, you could use a pseudo-random number generator with a fixed seed value.

Reply:
Added an option to use pseudo-random number with fixed seed value.